input/tests/measure/UrinarySymptomScoreChangeAfterBenignProstaticHyperplasiaFHIR/623a7f40-c265-4e42-a5c5-6f54e20b19df/CMS771FHIR-v1.3.000-DENEXFail-UrinaryRetentionDxUnconfirmed.json

```diff
@@ -0,0 +1,363 @@
+          "code": "active",
+          "display": "active",
+          "userSelected": true
+        } ]
+      },
+      "verificationStatus": {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+          "code": "confirmed",
+          "display": "Confirmed",
+          "userSelected": true
+        } ]
+      },
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code": "encounter-diagnosis",
+          "display": "Encounter Diagnosis"
+        } ]
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://hl7.org/fhir/sid/icd-10-cm",
+          "version": "2023",
+          "code": "N40.1",
+          "display": "Benign prostatic hyperplasia with lower urinary tract symptoms",
+          "userSelected": true
+        } ]
+      },
+      "subject": {
+        "reference": "Patient/623a7f40-c265-4e42-a5c5-6f54e20b19df"
+      },
+      "onsetPeriod": {
+        "start": "2024-12-14T15:00:00.000+00:00"
+      }
+    },
+    "request": {
+      "method": "POST",
+      "url": "Condition/55509232"
+    }
+  }, {
+    "fullUrl": "https://madie.cms.gov/Condition/55509255",
+    "resource": {
+      "resourceType": "Condition",
+      "id": "55509255",
+      "meta": {
+        "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-condition" ]
+      },
+      "clinicalStatus": {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+          "code": "active",
+          "display": "active",
+          "userSelected": true
+        } ]
+      },
+      "verificationStatus": {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+          "code": "unconfirmed",
+          "display": "Unconfirmed",
+          "userSelected": true
+        } ]
+      },
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code": "encounter-diagnosis",
+          "display": "Encounter Diagnosis"
+        } ]
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "version": "http://snomed.info/sct/731000124108/version/202303",
+          "code": "267064002",
+          "display": "Retention of urine (disorder)",
+          "userSelected": false
+        } ]
+      },
+      "subject": {
+        "reference": "Patient/623a7f40-c265-4e42-a5c5-6f54e20b19df"
+      },
+      "onsetPeriod": {
+        "start": "2024-12-31T00:00:00.000+00:00"
+      }
+    },
+    "request": {
+      "method": "POST",
+      "url": "Condition/55509255"
+    }
+  }, {
+    "fullUrl": "https://madie.cms.gov/Observation/55509233",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "55509233",
```

Comment:
A purely numeric ID will fail in HAPI probably need to change this to add a character to it